example/src/App.tsx

```diff
@@ -46,7 +48,9 @@ export default function App() {
             </p>
           </div>
         </div>
+      </div>
 
+      <div className="section">
```

Comment:
I think those should be custom react components and not classes - I think that woukd be the Tailwind way to do things